src/elements/content-preview/ContentPreview.js

```diff
@@ -1269,10 +1269,9 @@ class ContentPreview extends React.PureComponent<Props, State> {
         const onHeaderClose = currentVersionId === selectedVersionId ? onClose : this.updateVersionToCurrent;
 
         /* eslint-disable jsx-a11y/no-static-element-interactions */
-        /* eslint-disable jsx-a11y/no-noninteractive-tabindex */
         return (
             <Internationalize language={language} messages={messages}>
-                <div id={this.id} className={styleClassName} ref={measureRef} onKeyDown={this.onKeyDown} tabIndex={0}>
+                <div id={this.id} className={styleClassName} ref={measureRef} onKeyDown={this.onKeyDown}>
```

Comment:
@fsuarez-globant, does the `keydown` handling on this element still work without the tabindex? The left and right arrow keys should allow the user to move to the previous and next file in a collection, respectively.